tests/console/journalctl.pm

```diff
@@ -33,9 +33,9 @@ use constant {
     SEALING_DELAY => 10
 };
 
-# by default persistent journal is set for opensuse, except of leap 15.3+
+# Tumbleweed uses a persistent journal, Leap 15.3+ (except 15.3 AArch64 JeOS) inherits SLE's default
 sub has_default_persistent_journal {
-    return is_opensuse && (!is_leap('15.3+') || check_var('FLAVOR', 'JeOS-for-AArch64'));
+    return is_tumbleweed || (is_leap('15.3') && check_var('FLAVOR', 'JeOS-for-AArch64'));
```

Comment:
I guess you need to pass '=15.3'

Reply:
Yep, I fell into that trap again. It started the post_fail_hook which just wasted time...